### ISSUE TITLE
[FEA] Add a storage performance visualization

### ISSUE THREAD

**Issue by randerzander** [rapidsai/jupyterlab-nvdashboard#46]
In some workflows, we are heavily IO bound. If you watch the dask status page and see long `read_${format}` tasks, you might intuit that storage hardware is running slowly.

A nice way to expose storage performance to users could be via JupyterLab extension. I'm not sure if that would need to be part of this project or something separate, but GPUs starving for data is certainly something we need a way to observe.

**Comment by jacobtomlinson** [comment]
I totally agree this would be a useful thing to have. However I don't know if this is in scope for nvdashboard.

I wonder is a system dashboard showing memory usage, storage IO, etc would be useful as a separate project.

**Comment by jakirkham** [comment]
Have you looked at [glances]( https://nicolargo.github.io/glances/ )?

**Comment by jacobtomlinson** [comment]
I use the CLI mode a lot, but I always forget it has a web view too!
